common/app/views/support/HtmlCleaner.scala

```diff
@@ -896,8 +897,7 @@ case class AffiliateLinksCleaner(
         defaultOffTags,
         alwaysOffTags,
         tags,
-        publishedDate,
-      )
+      )(request)
```

Comment:
You might not need to explicitly pass the request here, since it's implicitly available on the `AffiliateLinksCleaner` class

Reply:
Yep seems to compile when removed, will make the change